tests/unit_and_functional/conftest.py

```diff
@@ -53,17 +55,18 @@ def setup_bodywork_test_project(
 ) -> Iterable[bool]:
     # SETUP
     try:
-        run(['git', 'init'], cwd=project_repo_location, check=True)
-        run(['git', 'add', '-A'], cwd=project_repo_location, check=True)
-        run(['git', 'commit', '-m', '"test"'], cwd=project_repo_location, check=True)
-        run(['mkdir', bodywork_output_dir], check=True)
-    except CalledProcessError as e:
-        raise RuntimeError(f'Cannot create test project Git repo - {e}.')
-    yield True
-    # TEARDOWN
-    run(['rm', '-rf', '.git'], cwd=project_repo_location)
-    run(['rm', '-rf', cloned_project_repo_location])
-    run(['rm', '-rf', bodywork_output_dir])
+        run(['git', 'init'], cwd=project_repo_location, check=True, encoding='utf-8')
+        run(['git', 'add', '-A'], cwd=project_repo_location, check=True, encoding='utf-8')
+        run(['git', 'commit', '-m', '"test"'], cwd=project_repo_location, check=True, capture_output=True, encoding='utf-8')
```

Comment:
This line is 117 character long - can you split it over two lines?

Reply:
Well spotted!